web-console/knikubevirt/vm-details/devices-and-resources/devices-and-respurces.md

```diff
@@ -0,0 +1,40 @@
+# VM details - Devices-and-resources
```

Comment:
I don't think we should include devices here. It could be a bit misleading b/c we have the idea of virtual hardware in the create flow.

Reply:
I'll check with Timas/Fabian

Reply:
See response in the next comment

Reply:
Thomas agreed that we should drop the "Devices"